lxd/instances_post.go

```diff
@@ -1240,15 +1240,6 @@ func instancesPost(d *Daemon, r *http.Request) response.Response {
 			}
 		}
 
-		if !clusterNotification {
```

Comment:
i dont think we want this commit, as we do not want to re-run the creation checks again during a redirection.

However please can you check whether `clusterNotification` is forgeable by a non-admin client?

Reply:
It's not forgeable; [client authentication will fail](https://github.com/canonical/lxd/blob/1e0b3535f2c0e0559205b019831bea7af1af21fb/lxd/daemon.go#L396) if the user passes `User-Agent: lxd-cluster-notifier`:
```
$ curl --cert client.crt --key client.key --insecure -H 'User-Agent: lxd-cluster-notifier' https://canaveral.cld.lan:8443/1.0
{"type":"error","status":"","status_code":0,"operation":"","error_code":403,"error":"Cluster notification isn't using trusted server certificate","metadata":null}
```
Dropping this commit.